doc/generic/pgf/pgfmanual-en-math-parsing.tex

```diff
@@ -203,8 +203,7 @@ \subsubsection{Commands}
 \end{command}
 
 \begin{command}{\pgfmathsetlengthmacro\marg{macro}\marg{expression}}
-    Defines \meta{macro} as the value of \meta{expression} \LaTeX{} \emph{in
-    points}.
+    Defines \meta{macro} as the value of \meta{expression} \emph{in points}.
```

Comment:
Or more clearly
```suggestion
    Defines \meta{macro} as the value of \meta{expression} in \TeX\ points (|pt|).
```
like line 220/219 in the same file below
https://github.com/pgf-tikz/pgf/blob/80475cd236b489fb8519ad280ac93b47e4e1f82f/doc/generic/pgf/pgfmanual-en-math-parsing.tex#L215-L221